src/app/loadout-builder/generated-sets/GeneratedSetSockets.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../variables.scss';
+
+.lockedItems {
```

Comment:
I'm generally a fan of new stuff being CSS modules, just a note for later

Reply:
Yeah same, this says its a new file even though its not it just got renamed and small modifications. There was a reason why it wasn't a module but I can't remember why. I might try convert it.

Reply:
Converted it over